local-environment/README.md

```diff
@@ -0,0 +1,79 @@
+
+# Local AEM Environment Setup with Docker
+
+This setup consists of a dispatcher connected by default with AEM publish and AEM author deployed via [gradle-aem-plugin](https://github.com/Cognifide/gradle-aem-plugin). Dispatcher runs on docker and its image is build based on [dispatcher/Dockerfile](dispatcher/Dockerfile).
+
+AEM author and publish instances need to be up.
+
+## Environment in details
+1. AEM author at [http://localhost:4502](http://localhost:4502)
+2. AEM publish at [http://localhost:4503](http://localhost:4503)
+3. Dispatcher available under three different domains:
+  * http://example.com -> which maps to `/content/example/website` content root on publish
+  * http://demo.example.com -> which maps to `/content/example/demo` content root on publish
+  * http://author.example.com -> which is proxy to the author instance
+
+## Configuration
+Most of the configuration steps are automated. However, there are three manual steps to make this setup fully operating:
+
+1. [Install docker](https://docs.docker.com/install/)
+2. Extend your `/etc/hosts` (`c:\Windows\System32\Drivers\etc\hosts` for Windows 10) with entries: 
+    ```bash
+    127.0.0.1       example.com
+    127.0.0.1       demo.example.com
+    127.0.0.1	author.example.com
+    ```
+3. Setup dispatcher flush on author for cache invalidation (make sure your author and publish instances are up)
```

Comment:
could be automated by additional gradle task `aemEnvConfigure` which will simply `aem.sync(aem.authors) { post("/etc/replication/.../author", mapOf("propName", "propValue"))` / to set correct configuration

Reply:
... of course; that should be a part of existing task if possible `aemEnvSetup`

Reply:
this thing will require `aemUp` ended / instance available

Reply:
OK, will work on it